src/crudeditor-lib/views/edit/workerSagas/save.js

```diff
@@ -136,7 +138,9 @@ export default function*({
             view: {
               name: VIEW_CREATE,
               state: {
-                predefinedFields: {} // TODO: build correct pre-filled instance.
+                predefinedFields: {
+                  ...(defaultNewInstance ? defaultNewInstance() : null)
```

Comment:
According to [documentation](https://github.com/OpusCapita/react-crudeditor/blob/master/README.md#definition-object-structure) **defaultNewInstance()** must be called with Search View state as an argument